cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/CombinedHBaseMetricsTable.java

```diff
@@ -100,7 +100,8 @@ private void deleteColumns(SortedMap<byte[], ? extends SortedMap<byte[], Long>>
         try {
           v2HBaseTable.delete(row, columns);
         } catch (Exception e) {
-          handleV2TableException(e, "delete");
+          handleV2TableException(e, String.format("Delete operation failed for table %s",
```

Comment:
you could just pass the name of the operation and the name of the table to the method...